openc3-cosmos-cmd-tlm-api/app/controllers/scopes_controller.rb

```diff
@@ -35,7 +35,16 @@ def index
 
   def create(update_model = false)
     return unless authorization('superadmin')
-    super(update_model)
+    model = @model_class.from_json(params[:json], scope: params[:scope])
+    if update_model
+      model.update
+      OpenC3::Logger.info("#{@model_class.name} updated: #{params[:json]}", scope: params[:scope], user: username())
+    else
+      model.create
+      model.deploy
```

Comment:
Deploy has two required arguments.   You can pass "." and {} for these like in openc3cli.
model.deploy(".", {})